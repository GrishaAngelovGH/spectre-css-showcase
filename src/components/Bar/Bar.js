const Bar = () => (
  <div className='bar'>
    <div className='bar-item tooltip' data-tooltip='25%' role='progressbar' style={{ width: '25%' }}>25%</div>
    <div className='bar-item tooltip' data-tooltip='15%' role='progressbar' style={{ width: '15%', background: '#817fe3' }}>15%</div>
    <div className='bar-item tooltip' data-tooltip='10%' role='progressbar' style={{ width: '10%', background: '#aaa9eb' }}>10%</div>
    <div className='bar-item tooltip' data-tooltip='15%' role='progressbar' style={{ width: '15%' }}>15%</div>
  </div>
)

export default Bar