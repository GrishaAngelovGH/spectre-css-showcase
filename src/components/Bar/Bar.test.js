import { render } from '@testing-library/react'

import Bar from './Bar'

test('should render Bar component', () => {
  const view = render(<Bar />)

  expect(view).toMatchSnapshot()
})