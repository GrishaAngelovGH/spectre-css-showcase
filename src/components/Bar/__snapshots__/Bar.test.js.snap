// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Bar component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bar"
      >
        <div
          class="bar-item tooltip"
          data-tooltip="25%"
          role="progressbar"
          style="width: 25%;"
        >
          25%
        </div>
        <div
          class="bar-item tooltip"
          data-tooltip="15%"
          role="progressbar"
          style="width: 15%; background: rgb(129, 127, 227);"
        >
          15%
        </div>
        <div
          class="bar-item tooltip"
          data-tooltip="10%"
          role="progressbar"
          style="width: 10%; background: rgb(170, 169, 235);"
        >
          10%
        </div>
        <div
          class="bar-item tooltip"
          data-tooltip="15%"
          role="progressbar"
          style="width: 15%;"
        >
          15%
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bar"
    >
      <div
        class="bar-item tooltip"
        data-tooltip="25%"
        role="progressbar"
        style="width: 25%;"
      >
        25%
      </div>
      <div
        class="bar-item tooltip"
        data-tooltip="15%"
        role="progressbar"
        style="width: 15%; background: rgb(129, 127, 227);"
      >
        15%
      </div>
      <div
        class="bar-item tooltip"
        data-tooltip="10%"
        role="progressbar"
        style="width: 10%; background: rgb(170, 169, 235);"
      >
        10%
      </div>
      <div
        class="bar-item tooltip"
        data-tooltip="15%"
        role="progressbar"
        style="width: 15%;"
      >
        15%
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
